### PR TITLE
Feature/gamma template Martí Malek

### PR DESCRIPTION
**Pull request by martimalek**

### CODE REVIEW

File: staff/marti-malek/gamma-template/about.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            <nav>
+                <a href="./home.html">Home</a>
+                <a href="./about.html">About</a>
+                <a href="./contact.html">Contact</a>
+            </nav>
+        </div>
+        <button><i class="fas fa-bars"></i></button>
+    </header>
+    <main>
+        <div class="about-t"><h1>About Gamma</h1></div>
```
Comment (maryammdot) [inline staff/marti-malek/gamma-template/about.html:24]: Are you giving styles to that div tag? If not, you don't need to add a class to it.

File: staff/marti-malek/gamma-template/about.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        <div class="about-grid">
+            <section class="about-first">
+                <div>
```
Comment (maryammdot) [inline staff/marti-malek/gamma-template/about.html:27]: Do you need that div? Maybe with the section tag is enough?

File: staff/marti-malek/gamma-template/contact.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                <a href="./contact.html">Contact</a>
+            </nav>
+        </div>
+        <button><i class="fas fa-bars"></i></button>
+    </header> 
+    <div class="contact-t"><h1>Contact Us</h1></div>
+    <main class="contact-grid">
+        <section class="contact-in">
+            <h4>Name</h4>
+            <input type="text" placeholder="Enter name">
```
Comment (maryammdot) [inline staff/marti-malek/gamma-template/contact.html:27]: input tags should be inside form tags

File: staff/marti-malek/gamma-template/style/style.sass [hunk 4]
```diff
@@ -0,0 +1,701 @@
...
+        border: 1px solid gray
+        border-radius: 0.25rem
+        height: 2rem
+        width: 3rem
+        margin-left: 1rem
+        margin-right: 1rem
+
+main
+    box-sizing: border-box
+    display: flex
```
Comment (maryammdot) [inline staff/marti-malek/gamma-template/style/style.sass]: Here you could use your %flex-wrap

File: staff/marti-malek/gamma-template/style/style.sass [hunk 5]
```diff
@@ -0,0 +1,701 @@
...
+    width: 100%
+    margin-top: 1rem
+    h1
+        color: white
+        padding: 1rem
+    ul
+        list-style: none
+        color: white
+    ul li:before
+        content: '✓ '
```
Comment (maryammdot) [inline staff/marti-malek/gamma-template/style/style.sass:141]: cool research of css!

File: staff/marti-malek/gamma-template/style/style.sass [hunk 6]
```diff
@@ -0,0 +1,701 @@
...
+            justify-self: left
+        p
+            grid-column: 1 / 2
+            grid-row: 3 / 4
+            margin: 0
+            margin-left: 4rem
+            font-size: 1.8rem
+            padding-right: 2rem
+            text-align: left
+        :nth-child(3)
```
Comment (maryammdot) [inline staff/marti-malek/gamma-template/style/style.sass:246]: Be carefull with the nth-child() in scalable projects

File: staff/marti-malek/gamma-template/style/style.sass [hunk 7]
```diff
@@ -0,0 +1,701 @@
+%img
```
Comment (maryammdot) [inline staff/marti-malek/gamma-template/style/style.sass]: cool use of inheritance! But in fact if you are not going to reuse the code, then it's not necessary.

File: staff/marti-malek/gamma-template/about.html (hunk 1 above)
Comment (martimalek) [reply staff/marti-malek/gamma-template/about.html:24]: I'm using the class in style.sass

File: staff/marti-malek/gamma-template/about.html (hunk 2 above)
Comment (martimalek) [reply staff/marti-malek/gamma-template/about.html:27]: I'm referring to it in style.sass

File: staff/marti-malek/gamma-template/style/style.sass (hunk 6 above)
Comment (martimalek) [reply staff/marti-malek/gamma-template/style/style.sass:246]: Should I change something?

File: staff/marti-malek/gamma-template/style/style.sass (hunk 7 above)
Comment (martimalek) [reply staff/marti-malek/gamma-template/style/style.sass]: True, i've changed it!

File: staff/marti-malek/gamma-template/style/style.sass (hunk 5 above)
Comment (martimalek) [reply staff/marti-malek/gamma-template/style/style.sass:141]: Thx!

File: staff/marti-malek/gamma-template/style/style.sass (hunk 4 above)
Comment (martimalek) [reply staff/marti-malek/gamma-template/style/style.sass]: Didn't see it! Thx!

File: staff/marti-malek/gamma-template/contact.html (hunk 3 above)
Comment (martimalek) [reply staff/marti-malek/gamma-template/contact.html:27]: Ok, i've changed it. Thx!

File: staff/marti-malek/gamma-template/style/style.sass (hunk 6 above)
Comment (maryammdot) [reply staff/marti-malek/gamma-template/style/style.sass:246]: No, but keep it in mind 🤓

**Pull request merged**